kafkaesk/consumer.py

```diff
@@ -425,25 +434,25 @@ async def healthy(self) -> None:
     # Event handlers
     async def on_partitions_revoked(self, revoked: typing.List[aiokafka.TopicPartition]) -> None:
         if revoked:
+            async with self._processing:
+                if self._auto_commit:
```

Comment:
should this be `if not self._auto_commit` ?

Reply:
if `auto_commit == False` it means that all the commiting is done in the app side.